docs/onnx-conversion.md

```diff
@@ -1,75 +1,251 @@
-# ONNX Conversion for SPLADE++
+# End to End ONNX Conversion for SPLADE++ Ensemble Distil
 This MD file will describe steps to convert particular PyTorch models (i.e., [SPLADE++](https://doi.org/10.1145/3477495.3531857)) to ONNX models and options to further optimize compute graph for Transformer-based models. For more details on how does ONNX Conversion work and how to optimize the compute graph, please refer to [ONNX Tutorials](https://github.com/onnx/tutorials#services).
 
+All scripts are available for reference under in the following directory:
+```
+src/main/python/onnx
+```
+
+The following tutorial will be using the scripts available in the above directory directly.
+
 ## Setups
-- pip requirements:
-    ```
-    onnx                          1.13.1
-    onnxoptimizer                 0.3.10
-    onnxruntime                   1.11.1
-    ```
-- SPLADE [Git repo](https://github.com/naver/splade):
-   ```bash
-   git clone https://github.com/naver/splade.git
-   ```
+
+pip requirements:
+```bash
+pip install torch transformers onnx onnxruntime onnxoptimizer
```

Comment:
Are we ok with installing any version or do we want to keep a defined version as it was before?

Reply:
Any version should be okay, tested with it on fixed versions that were previously provided and opset 14 isnt supported for the provided onnx. Also I wasn't able to find onnxruntime 1.11.1

Reply:
Maybe you can put the versions you were using on yours as a secondary one, something like 
`we tested with the following versions 'pip install... ' but we recommend trying first with the latest one from the previous command`

Reply:
Yes, sounds good